.github/workflows/e2e-tests.yaml

```diff
@@ -61,6 +61,15 @@ jobs:
         kubectl -n triggermesh wait --timeout=1m --for=condition=Available deployments.apps/triggermesh-controller
         kubectl -n triggermesh wait --timeout=1m --for=condition=Available deployments.apps/triggermesh-webhook
 
+    - name: Install Ginkgo
+      run: go install github.com/onsi/ginkgo/ginkgo
+
+    - name: Export KUBECONFIG path
+      run: echo "KUBECONFIG=${HOME}/.kube/config" >> $GITHUB_ENV
```

Comment:
AFAIK it is not possible to interpolate `HOME` inside the `env:` of workflow steps because it is not part of GitHub Actions' "default environment variables" (such as `GITHUB_WORKSPACE`, etc.), so I used the `$GITHUB_ENV` mechanism documented at https://docs.github.com/en/actions/learn-github-actions/environment-variables to achieve that.

Similar topics:
- https://github.community/t/how-to-use-env-context/16975
- https://github.community/t/env-vars-are-not-expanded-causing-dry-violations/16550